src/Skyhop.Mail/RazorViewToStringRenderer.cs

```diff
@@ -16,15 +16,26 @@
 
 namespace Skyhop.Mail
 {
-    // Code from: https://github.com/aspnet/Entropy/blob/master/samples/Mvc.RenderViewToString/RazorViewToStringRenderer.cs
```

Comment:
Great, while inspired on this, I don't think one can compare these two versions anymore :)